lib/hypervisor/hv_kvm/netdev.py

```diff
@@ -189,6 +189,9 @@ def OpenTap(name="", features=None):
 
     tapfds.append(tapfd)
 
+    if ( _ == 0 and name == "" ):
```

Comment:
Since we are now checking the value of `_`, it should get a proper variable name such as `q_idx`. `_` is typically used for throw-away variables whose value we don't care about.

Then again, you probably don't care at all if this is the first iteration, just checking if `name` is blank should be enough.

Also the parentheses here are not necessary :)

Reply:
I think `if name == ""` is sufficient here.